### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Corpus has been renamed to testsuite.

File: disk.mayhemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+project: ntfs-rust
```
Comment (xansec) [inline disk.mayhemfile:1]: Tests has been renamed to testsuite.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: disk.mayhemfile (hunk 2 above)
Comment (evanrichter) [reply disk.mayhemfile:1]: I removed this part in favor of using the default _/testsuite/_ in the docker image

File: Dockerfile (hunk 1 above)
Comment (evanrichter) [reply Dockerfile:1]: fixed, thanks!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**